tools/autograd/derivatives.yaml

```diff
@@ -138,6 +138,12 @@
 - name: alias(Tensor self)
   self: grad
 
+- name: any(Tensor self)
```

Comment:
Yes, I think this is necessary.

> I was hitting type mismatch error (Variable[CPUByteType] vs CPUByteType) in TensorIterator without this change, because codegen is not unwrapping the input Tensor for all(*) without this line.